contracts/Tokens/Prime/README.md

```diff
@@ -0,0 +1,334 @@
+
+There is a variable named `totalScoreUpdatesRequired` to track how many scores updates are pending. This is for tracking purposes and visibility to the community.
+
+## Calculate APR associated with a Prime market and user
+
+The goal is to offer a view function that allows the [Venus UI](https://app.venus.io) to show an estimation of the APR associated with the Prime token and the borrow/supply position of a user.
+
+The steps to perform this calculation are:
+
+1. calculate the income per block (see below) // TODO link
+2. get from the [ProtocolShareReserve](https://github.com/VenusProtocol/protocol-reserve/blob/develop/contracts/ProtocolReserve/ProtocolShareReserve.sol) the percentage associated with Prime (`distributionTarget` where the destination is `address(Prime)` and `schema` is `SPREAD_PRIME_CORE`
+3. calculate the income per block sent to Prime (1 times 2), sum the funds provided by the Prime Liquidity Provider contract, and multiply it by the number of blocks in one year
+4. multiply (3) by the user score in that market, and divide it by the sum of scores in that market. This is the extrapolation of income generated by Prime to this user
+5. split (4) proportional to the (capped) borrow and supply amounts of the user in that market at that moment, and divide these numbers to calculate the APR
+
+**Example:**
+
+1. Income per block 0.0003 USDT
+2. Percentage to Prime: 0.1 (10%)
+3. Income per block sent to Prime: 0.0003 * 0.1 = 0.00003 USDT. Multiply per 10512000 blocks/year = 315.36 USDT
+4. Assuming the user score for USDT: 3, and the sum of scores for USDT: 10, then we would have 94.608 USDT (yearly income for this user, generated by Prime)
+5. Assuming the user has the following positions:
+    1. borrow: 30 USDT. Let's say it's capped at 15 USDT, so we'll consider 15 USDT
+    2. supply: 10 USDT. Let's say it's also capped at 15 USDT, so we'll consider 10 USDT
+6. Allocating the rewards (94.608 USDT), considering these capped versions, we would have:
+    1. borrow: 94.608 * 15/25 = 56.76 USDT
+    2. supply: 94.608 * 10/25 = 37.84 USDT
+7. Calculating the APR with these allocations, we would have:
+    1. borrow: 56.76/30 = 1.89 = 189%
+    2. supply: 37.84/10 = 3.78 = 378%
+
+Only part of the supplied and borrowed amounts (the capped amounts) are really "working" to increase the prime rewards. The rest of the supplied or borrowed amounts do not generate extra rewards. In the example, if the user supplies more USDT, they won't generate more rewards (because the supply amount to be considered is capped at 15 USDT). So, it would make sense that the supply APR would decrease if they supply more USDT.
```

Comment:
```suggestion
Only part of the supplied and borrowed amounts (the capped amounts) are actually "working" to increase the prime rewards. The rest of the supplied or borrowed amounts do not generate extra rewards. In the example, if the user supplies more USDT, they won't generate more rewards (because the supply amount to be considered is capped at 15 USDT). So, it would make sense that the supply APR would decrease if they supply more USDT.
```